doc/misc/npm-scripts.md

```diff
@@ -139,6 +139,23 @@ then the user could change the behavior by doing:
 
     npm config set foo:port 80
 
+### conditionals and npmrc variables
+
+You can use bash conditionals to determine, for example, what your 
```

Comment:
This won't work on Windows. We can still have these docs, but they should note that.

Reply:
Good point! I've added a note around operating systems, hopefully that's what you had in mind. Thanks